nn/normalization.py

```diff
@@ -183,3 +184,44 @@ def normalize(a: nn.Tensor, *, axis: Union[nn.Dim, Sequence[nn.Dim]], epsilon: f
 
   mean, variance = nn.moments(a, axis=axis)
   return (a - mean) * nn.rsqrt(variance + epsilon)
+
+
+class Normalize(nn.Module):
+  """
+  Wrapper around normalize with additional scale and bias
+  """
+
+  def __init__(self,
+               axis: Union[nn.Dim, Sequence[nn.Dim]],
+               epsilon: float = 1e-6,
+               scale: bool = True, bias: bool = True):
+    """
+
+    :param axis: axis over which normalization is computed
+    :param epsilon: epsilon for numerical stability
+    :param scale: whether to include a trainable scale
+    :param bias: whether to include a trainable bias
+    """
+    super(Normalize, self).__init__()
+    self.epsilon = epsilon
+    self.axis = axis
+    if isinstance(axis, nn.Dim):
+      axis = [axis]
+    self.scale = nn.Parameter(shape=axis, trainable=True) if scale else None
```

Comment:
You don't need the `trainable` argument. The code should be consistent to other code.

Reply:
Why do I not need it? Its default is not `True`.

Reply:
It is (on RETURNN side). And in any case, it should be consistent to other code.

Reply:
Ah okay, I did not know that.